src/components/ScrollArea/ScrollArea.tsx

```diff
@@ -52,6 +55,31 @@ const ScrollArea = ({
       } else if (root.style.getPropertyValue('--bottom-fog-opacity') !== '1') {
         root.style.setProperty('--bottom-fog-opacity', '1');
       }
+
+      // viewport.scrollTop < 0
+      //   ? setTopOverscroll(0 - viewport.scrollTop)
+      //   : setTopOverscroll(0);
+
+      // const threshold = 96;
+
+      // if (viewport.scrollTop <= -threshold) {
+      //   setIsPullingDown(true);
+      // }
+
+      // if (
+      //   viewport.scrollTop > -threshold &&
+      //   viewport.scrollTop + viewport.clientHeight <
+      //     viewport.scrollHeight + threshold
+      // ) {
+      //   setIsPullingDown(false);
+      // }
+
+      // if (
+      //   viewport.scrollTop + viewport.clientHeight >=
+      //   viewport.scrollHeight + threshold
+      // ) {
+      //   viewport.style.background = 'green';
+      // }
```

Comment:
kann raus?

Reply:
Wollte eigentlich #70 in der selben Branch machen und hatte hiermit angefangen. Mache das als nächstes separat, aber hast Recht, ich kann das eben stashen

Reply:
Kann drin bleiben 🏁